staging/src/kubevirt.io/client-go/api/v1/schema.go

```diff
@@ -691,6 +691,9 @@ type Volume struct {
 	// Must be a DNS_LABEL and unique within the vmi.
 	// More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
 	Name string `json:"name"`
+	// Hotpluggable indicates whether the volume can be hotplugged and hotunplugged.
+	// +optional
+	Hotpluggable bool `json:"hotpluggable,omitempty"`
```

Comment:
The `Hotpluggable` attribute does not seem to be applicable for all possible volume sources. E.g. it does not make much sense with `ContainerDiskSource` and some others. So I guess there should be validation that prevents from setting it to `true` with unsupported types.

But on the other hand, since it is valid only for `PersistentVolumeClaim` and `DataVolume` maybe it makes sense to have it as part of `PersistentVolumeClaimVolumeSource` and `DataVolumeSource`? Then the validation will not be needed.

Reply:
Nice catch! It will be great to eliminate unnecessary validations.

Reply:
It seems that we can simply make `Hotpluggable` as a part of `DataVolumeSource`. However, for `PersistentVolumeClaimVolumeSource` I am not sure how, since [it's defined in K8s core v1](https://git.io/JlNXV). If is a breaking change, it we provide a new struct wrapping it in KubeVirt. Should we go this way or is there any better way?

Reply:
Right, probably I havent considered the case with `PersistentVolumeClaimVolumeSource`...

Reply:
What about making a wrapper struct?
```go
type PersistentVolumeClaimVolumeSource struct {
    v1.PersistentVolumeClaimVolumeSource
    Hotpluggable bool
}
```
Then no need to copy from k8s pkg.

Reply:
This was my first thought as well, why not do a wrapper struct that way we are not duplicating and possibly getting out of sync with the k8s core struct.